.github/workflows/ci-build.yml

```diff
@@ -0,0 +1,64 @@
+name: CI Build
+
+on:
+  push:
+    branches:
+      - master
+      - 'v*'
+  pull_request: {}
+  schedule:
+    - cron: '0 3 * * *' # daily, at 3am
+
+jobs:
+  test:
+    name: Tests
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 10.x
+      - name: install dependencies
+        run: yarn install
+      - name: lint:js
+        run: yarn lint:js
+      - name: lint:hbs
+        run: yarn lint:hbs
+      - name: test
+        run: yarn test
+
+  try-scenarios:
+    name: ${{ matrix.ember-try-scenario }}
+
+    runs-on: ubuntu-latest
+
+    needs: test
+
+    strategy:
+      fail-fast: true
+      matrix:
+        ember-try-scenario:
+          - ember-lts-2.18
+          - ember-lts-3.4
+          - ember-lts-3.8
+          - ember-lts-3.12
+          - ember-lts-3.16
+          - ember-lts-3.20
+          - ember-lts-3.24
```

Comment:
This adds 3.24, but I don't see if 3.24 is in ember-try config. Can you double check?

Reply:
This was added in https://github.com/emberjs/ember-render-modifiers/pull/30